<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="mytests.OpenCartTest" tests="2" skipped="0" failures="0" errors="0" timestamp="2023-05-20T11:07:37" hostname="naveenautomationlabss-MacBook-Pro.local" time="1.166">
  <properties/>
  <testcase name="logoTest" classname="mytests.OpenCartTest" time="0.053"/>
  <testcase name="forgotPwdLinkExistTest" classname="mytests.OpenCartTest" time="0.037"/>
  <system-out><![CDATA[open cart - logo test
open cart - forgot pwd test
]]></system-out>
  <system-err><![CDATA[com.gurock.testrail.APIException: TestRail API returned HTTP 400("Field :case_id is not a valid ID.")
	at com.gurock.testrail.APIClient.sendRequest(APIClient.java:296)
	at com.gurock.testrail.APIClient.sendPost(APIClient.java:132)
	at com.manager.TestRailManager.addResultForTestCase(TestRailManager.java:27)
	at base.BaseTest.addResultsToTestRail(BaseTest.java:60)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:683)
	at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:658)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:642)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
]]></system-err>
</testsuite>
